### PR TITLE
Better mobile nav bar; Hanabi Live is now a link in header

### PR DESCRIPTION
**Pull request by padiwik**
See #1309

### CODE REVIEW

File: server/src/views/main.tmpl [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
   <!-- Header -->
   <header id="header">
-    <h1>Hanabi Live</h1>
+    <h1><a href="/">Hanabi Live</a></h1>
```
Comment (Zamiell) [inline server/src/views/main.tmpl:7]: it should't be on main because that will fuck up the user

Comment (padiwik) [reply server/src/views/main.tmpl:7]: why? it's just a reload

Comment (Zamiell) [reply server/src/views/main.tmpl:7]: if the user is on a slow computer or a slow internet connection, one stray misclick will cause a frustrating 30 second period where they cannot do anything anymore. that is terrible

**Comment by Zamiell** [comment]
confirmed on both main and other sites, looks good

**Comment by Zamiell** [comment]
padi i think the easiest way is to just make a breakpoint that removes the padding on the buttons

**Comment by padiwik** [comment]
did u mean to comment on valetta's issue?

**Comment by Zamiell** [comment]
yes oops

**Pull request merged**